config/config.go

```diff
@@ -55,6 +65,7 @@ type ServerConfig struct {
 	DbSslCertContents     string `ini:"db_sslcert_contents"`
 	DbSslKey              string `ini:"db_sslkey"`
 	DbSslKeyContents      string `ini:"db_sslkey_contents"`
+	DbUseIamAuth          bool   `ini:"db_use_iam_auth"`
```

Comment:
We'll also have to add this setting to the `getDefaultConfig()` logic in `read.go`, so we support setting it through an environment variable (which is the case when deploying as a container).

As a side note, I was trying to understand whether "IAM" is a generic enough term, but looks like at least Google Cloud SQL also uses that terminology (https://cloud.google.com/sql/docs/postgres/authentication) so I think we could keep that naming and assume we mean it in a generic way.